.github/workflows/automatic_pull_request_review.yml

```diff
@@ -42,12 +42,12 @@ jobs:
       run: bundle exec pod install
 
     - name: Build and Test
-      run: bundle exec fastlane build_and_test
+      run: bundle exec fastlane buildAndTestLane --disable_runner_upgrades
```

Comment:
@nkhanh44 FYI, we don't commit `FastlaneRunner` executable file to git, so we don't need to disable its upgrade. On CI, it works well without `--disable_runner_upgrades`

Reply:
Thank you for the information, I updated it at https://github.com/nimblehq/ios-templates/pull/395/commits/3e288e5aa62e42dd31a650742d9d230a168ae9f4 😄.